M2/Macaulay2/m2/hypertext.m2

```diff
@@ -363,6 +364,14 @@ style Hypertext := true >> o -> x -> (
     append(x,"style"=>str)
     )
 
+-- hijacked "hypertext"
```

Comment:
If you're going to repurpose hypertext for this, I don't think there's a need for comments like this. A mysteriously commented `fixup` won't help anyone. Also, is the typical value `Hypertext`?

Reply:
the typical value was always "Hypertext". I'll remove the offending comment...

Reply:
Is there a reason why you can't bring all the `hypertext` methods defined in `html.m2` here? Would be nice to have a section in `hypertext.m2` for the ways internal M2 types are converted to hypertext objects defined above.